declarations/Open Terms Archive.history.json

```diff
@@ -4,13 +4,23 @@
       "fetch": "https://opentermsarchive.org/privacy-policy",
       "select": ".TextContent_textContent__ToW2S",
       "validUntil": "2024-03-18T18:30:09Z"
+    },
+    {
+      "fetch": "https://opentermsarchive.org/privacy-policy",
+      "select": ".textcontent",
+      "validUntil": "2024-04-19T12:30:10Z"
     }
   ],
   "Imprint": [
     {
       "fetch": "https://opentermsarchive.org/legal-notice",
       "select": ".TextContent_textContent__ToW2S",
       "validUntil": "2024-03-18T18:30:09Z"
+    },
+    {
+      "fetch": "https://opentermsarchive.org/legal-notice",
+      "select": ".textcontent",
+      "validUntil": "2024-04-16T18:30:11.000Z"
```

Comment:
Why do the `validUntil` dates differ for both documents when the change is a consequence of https://github.com/OpenTermsArchive/opentermsarchive.org/pull/318?

Reply:
It's a mistake, I've corrected it.

Reply:
https://github.com/OpenTermsArchive/opentermsarchive.org/pull/318 was merged on April 19th. How can the date of the switch here be April 16th? 🤔